drivers/overlay/ov_serf.go

```diff
@@ -146,6 +146,7 @@ func (d *driver) processQuery(q *serf.Query) {
 		return
 	}
 
+	logrus.Debugf("Sending peer query resp %v", fmt.Sprintf("%s %s %s", peerMac.String(), net.IP(peerIPMask).String(), vtep.String()))
```

Comment:
why `%v` is you are actually formatting a string as parameter, should be `%s`

`peerIPMask` is already of type `net.IP` no need to cast

net.HardwareAddr, net.IPMask and net.IP implement `String() string`, you do not need to call it.